Update FAQ (#343)
The-Strategy-Unit/nhp_project_information: user_guide/FAQ.qmd

## Are there plans for other baseline years?

> With the latest release 2024/5 data is now available. The previous year 2023/4 is also still avaialable to support users who have set and run models already with 2023/4 data. It is essential that you use the baseline data that is appropriate for the modelling stage that you are at- changing to earlier or later baseline dates will severely impact progress with business planning by invalidating modelling results.

Reviewer: ```suggestion
With the latest release 2024/5 data is now available. The previous year 2023/4 is also still avaialable to support users who have set and run models already with 2023/4 data. It is essential that you use the baseline data that is appropriate for the modelling stage that you are at, because changing to a different baseline year will severely impact progress with business planning by invalidating modelling results.
```
